libtsuba/src/RDGManifest.h

```diff
@@ -74,15 +80,20 @@ class KATANA_EXPORT RDGManifest {
   RDGManifest NextVersion(
       uint32_t num_hosts, uint32_t policy_id, bool transpose,
       const RDGLineage& lineage) const {
+    katana::RDGVersion next_version(version_.numbers_, version_.branches_);
```

Comment:
Why not have a copy constructor on RDGVersion?

Reply:
Good catch. It was very old code before the copy constructor.